meteor/client/ui/globals/keyboardGlobals.d.ts

```diff
@@ -0,0 +1,14 @@
+declare type KeyboardLayoutMap = Map<string, string>
+
+declare type KeyboardLayoutEvents = 'layoutchange'
+
+declare interface Keyboard {
+	getLayoutMap(): Promise<KeyboardLayoutMap>
+	addEventListener(type: KeyboardLayoutEvents, listener: EventListener): void
+	removeEventListener(type: KeyboardLayoutEvents, listener: EventListener): void
+	lock(): Promise<undefined>
```

Comment:
It might make sense to allow the optional `keyCodes` parameter to avoid confusion in the future.

Reply:
I was debating back and forth if I should include it or not and ended up not including it. But I have no arguments for why it shouldn't be there so it's been updated.